### PR TITLE
Upgrade Qt from 5.10 to 5.12

### LINKED ISSUE #70: Update Qt5
Qt 5.11 is no longer available for download. Upgrade to [5.12](http://download.qt.io/archive/qt/5.12/5.12.10/single/).

### PR DESCRIPTION
**Pull request by brobeson**
5.10 is no longer available for download. The next closest release is
5.12. I chose 5.12.10 to get the latest bug fixes.

Closes #70

### CODE REVIEW

File: projects/qt5.cmake [hunk 1]
```diff
@@ -171,7 +169,6 @@ function(build_qt5)
     SOURCE_DIR ${NULL_DIR} CONFIGURE_COMMAND "" BUILD_COMMAND ""
     INSTALL_COMMAND
       ${CMAKE_COMMAND} -E make_directory ${STAGE_DIR}/share/qt5 &&
-      ${CMAKE_COMMAND} -E copy ${SOURCE_DIR}/LGPL_EXCEPTION.txt ${STAGE_DIR}/share/qt5 &&
```
Comment (bnhhughes) [inline projects/qt5.cmake:174]: Does this file no longer exist in 5.12.10?

**Review by bnhhughes — APPROVED** [review]

File: projects/qt5.cmake (hunk 1 above)
Comment (swflb) [reply projects/qt5.cmake:174]: @brobeson what is the reason for removing this line?

**Review by swflb — APPROVED** [review]
just wondering why removing copying of a license file. perhaps a different one needs copied intead?

File: projects/qt5.cmake (hunk 1 above)
Comment (brobeson) [reply projects/qt5.cmake:174]: The file doesn't exist in 5.12.10, so this command causes the build to fail.

**Pull request merged**